### ISSUE TITLE
Facebook Page Plugin widget stream missing

### ISSUE THREAD

**Issue by juliasydnor** [Automattic/jetpack#13717]
#### Steps to reproduce the issue

1. Go to 'WP Admin > Appearance > Widgets'
2. Add a Facebook Page Plugin widget to any widget area.
3. Enable `Show Stream` in the widget settings.
4. Save the widget and view it on the live site.

#### What I expected

I expected the widget to display a stream of recent posts.

#### What happened instead

The widget only shows the cover photo (if enabled), and does not show a stream of recent posts. It was showing the stream until earlier today (first it disappeared only in Safari, and then Chrome and Firefox about 15 minutes later).

**Screenshots**

![Screen Shot on 2019-10-10 at 17:54:58](https://user-images.githubusercontent.com/29981758/66609454-1790f900-eb87-11e9-92af-30c4822d515e.png)
![Screen Shot on 2019-10-10 at 17:54:43](https://user-images.githubusercontent.com/29981758/66609461-1d86da00-eb87-11e9-881c-81e80395a5aa.png)

**Comment by jeherve** [comment]
Good catch! It would seem that Facebook has now deprecated this attribute:

> The attribute data-show-posts is deprecated. Please use the attribute tabs/data-tabs and use the value timeline to show posts from the Page's timeline.

PR incoming.
